ManiVault/src/private/HelpManager.cpp

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later 
+// A corresponding LICENSE file is located in the root directory of this source tree 
+// Copyright (C) 2023 BioVault (Biomedical Visual Analytics Unit LUMC - TU Delft) 
+
+#include "HelpManager.h"
+
+#include <Application.h>
+
+#include <QDesktopServices>
+
+using namespace mv::gui;
+using namespace mv::util;
+
+#ifdef _DEBUG
+    //#define HELP_MANAGER_VERBOSE
+#endif
+
+namespace mv
+{
+
+HelpManager::HelpManager() :
+    AbstractHelpManager(),
+    _showLearningCenterAction(this, "Learning center"),
+    _toDiscordAction(this, "Join us..."),
```

Comment:
"Join us..." sounds very ominous... maybe we can use [font-awsome discord icon](https://fontawesome.com/icons/discord) directly?

Reply:
I thought about that too, I had some problems showing the discord icon with FontAwesome. Perhaps we can call it: join us on Discord?

Reply:
That's also good